README.md

```diff
@@ -48,6 +48,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 - [Vitesse](https://github.com/antfu/vitesse) - Opinionated Vite starter template.
 - [vite-vue3-tailwind-starter](https://github.com/web2033/vite-vue3-tailwind-starter) - Starter template with Vue 3, Vue Router and Tailwind CSS.
+- [vite-ts-tailwind-starter](https://github.com/Uninen/vite-ts-tailwind-starter) - Opinionated starter template with TypeScript, Tailwind CSS, and Cypress.io e2e tests, CI configured for GitLab + GitHub.
```

Comment:
Looks cool, but can you make the description a little bit shorter? Thanks.

Reply:
Sure. Updated it from 120 to 72 chars. Ok now?

Reply:
Thanks!